webapp-django/crashstats/topcrashers/views.py

```diff
@@ -104,8 +105,19 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
-            # Number of startup crashes.
-            hit['startup_percent'] = 0
+            # Number of crashes happening during startup. This is defined by
+            # the software, as opposed to the next methid which relies on
+            # the uptime of the software.
+            hit['startup_count'] = 0
+
+            sig_startup = hit['facets']['startup_crash']
+            for row in sig_startup:
+                if row['term'] in ('T', '1'):
+                    hit['startup_count'] = row['count']
```

Comment:
This should be `+=`, right? There could be cases where there are both `T` and `1`.

For example, I've seen:
```
{
	"count": 496,
	"term": "nvd3d9wrapx.dll@0x3f47",
	"facets": {
		"platform": [{
			"count": 496,
			"term": "Windows NT"
		}],
		"startup_crash": [{
			"count": 321,
			"term": "1"
		}, {
			"count": 162,
			"term": "T"
		}]
	}
},
```